docs/development.md

```diff
@@ -37,6 +37,26 @@ Start an interactive console in a container, run:
 docker compose run --rm -u "$UID:$GID" --service-ports dev bash
 ```
 
+### Start Dev Container inspecting metrics
+
+When we would like to inspect metrics of our services, there have to be made adjustement under `dev` service.
```

Comment:
Maybe you can add it into docker compose, comment it out with a note what it does / when to enable it?

Reply:
I will leave this in documentation and add commented out code with explanation.